app/models/user.rb

```diff
@@ -5,20 +5,26 @@
 #  id                     :bigint(8)        not null, primary key
 #  email                  :string           default(""), not null
 #  encrypted_password     :string           default(""), not null
-#  reset_password_token   :string
-#  reset_password_sent_at :datetime
+#  full_name              :string
+#  oauth_expires_at       :string
+#  oauth_token            :string
+#  provider               :string
 #  remember_created_at    :datetime
+#  reset_password_sent_at :datetime
+#  reset_password_token   :string
+#  uid                    :string
 #  created_at             :datetime         not null
 #  updated_at             :datetime         not null
-#  uid                    :string
-#  provider               :string
-#  full_name              :string
-#  oauth_token            :string
-#  oauth_expires_at       :string
+#
+# Indexes
+#
+#  index_users_on_email                 (email) UNIQUE
+#  index_users_on_reset_password_token  (reset_password_token) UNIQUE
 #
 
 class User < ApplicationRecord
-  has_many :projects
+  has_many :workspaces
```

Comment:
```ruby
dependent: :delete_all
```

Reply:
@BogdanBusko where is dependent destroy? I'm writing again this comment